config/config.default.yaml

```diff
@@ -102,6 +102,7 @@ electricity:
   max_hours:
     battery: 6
     H2: 168
+    iron-air battery: 100
```

Comment:
I would keep this separate from the max_hour implementation for now, at least until the other PR is merged. It will have to be adjusted then when merging it